docs/reference/requirement-sets/image-coercion-requirement-sets.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Image Coercion requirement sets
+description: ''
```

Comment:
Suggest that you populate the description attribute, as it helps with SEO.

Reply:
@kbrandl Good call. Side-note: most of these pages don't have descriptions, but that's a larger work item we can tackle outside of this PR.